spec/helpers.lua

```diff
@@ -1910,6 +1910,47 @@ luassert:register("assertion", "cn", assert_cn,
                   "assertion.cn.positive")
 
 
+--- Assertions to check whether a value exists in a table
+-- @function has_value
+-- @param table
+-- @param value
+-- @return exists?
+-- @usage
+-- assert.has_value({"foo"}, "foo") -- true
+-- assert.not_has_value({"foo"}, "bar") -- true
+local function assert_has_value(state, args)
+    assert(type(args[1]) == "table",
+      "Expected first argument to be a table")
+
+    for _, v in pairs(args[1]) do
+      if v == args[2] then
+        return true
+      end
+    end
+    
+    return false
+end
+
+say:set("assertion.has_value.negative", [[
+Expected table to have the given value.
+Passed in
+%s
+Expected value
+%s
+]])
+say:set("assertion.has_value.positive", [[
+Expected table to not have the given value.
+Passed in
+%s
+Expected value
+%s
+]])
+
+luassert:register("assertion", "has_value", assert_has_value,
```

Comment:
```suggestion
luassert:register("assertion", "contains", assert_has_value,
```

Might be a better name IMO.

Reply:
https://github.com/Kong/kong/blob/d6202c22809a0a4f138cb9d58a4631f1b3de6ad5/spec/helpers.lua#L1467-L1502

There is a function with the same name, but it is only used for strings (regex). `has_value` is expected to be used for other types, such as number.

Reply:
In that case, can we make `contains` compatible with different value types?